velox/exec/tests/TaskTest.cpp

```diff
@@ -1184,7 +1184,7 @@ DEBUG_ONLY_TEST_F(TaskTest, liveStats) {
   EXPECT_EQ(32 * numBatches, operatorStats.outputBytes);
   EXPECT_EQ(3 * numBatches, operatorStats.outputPositions);
   EXPECT_EQ(numBatches, operatorStats.outputVectors);
-  EXPECT_EQ(1, operatorStats.finishTiming.count);
+  EXPECT_EQ(2, operatorStats.finishTiming.count);
```

Comment:
Is there a way to check `isBlockedTiming` e.g. in HashProbe?

Reply:
yes. I just added it to this test.
```
for (auto i = 0; i < numBatches; ++i) {
    ...
    EXPECT_EQ(2 * (i + 1), operatorStats.isBlockedTiming.count);
```
Sorry I didn't add it until now. I was blocked by the build issues for a long time.